### PR TITLE
Update dappnode.md

### PR DESCRIPTION
**Pull request by alexpeterson91**
## What

update store links to current domains, edit some some instructions and links to UI with updated data, fixed permalinks to our docs etc, remove outdated data, and removed the outdated section on how to install Dappnode replaced it with a link to our updated user-guide since the instructions there are out of date by roughly 7-8 months now.

## Refs

I'm Voss btw original author of most of this page and the Gnosis Wagyu port described within, core team @dappnode ;) 

- (this project accepts pull requests related to open issues)
- (if suggesting a new feature or change, please discuss it in an issue first)
- (if fixing a bug, there should be an issue describing it with steps to reproduce)
- (only fixing a minor bug - broken link, typo - an issue is not needed)
- (please link to the issue here)

### CODE REVIEW

File: docs/node/tools/dappnode.md [hunk 1]
```diff
@@ -145,29 +143,5 @@ In case you need some xDai for transaction fees you can get some from the [offic
 4. Your deposit file will be validated and list the number of validator deposits you are making and the required GNO to deposit. Click `Deposit` to continue.
 5. Check that you understand the risks and ensure you are interacting with the correct contract before proceeding.
 6. Click `Ok` and confirm the transaction in your wallet to complete the deposit.
+7. Our proxy smart contract will deposit the GNO(s) to your validators! YOU control the private keys, YOU control the withdrawal key(s)... these validators are now **yours**. Take good care of them!
 
-7. Our proxy smart contract will deposit the 4 GNO to your validators! YOU control the private keys, YOU control the withdrawal key... these validators are now **yours**. Take good care of them!
-
-:::tip DAppNode <\> Gnosis Chain Incentive Program
```
Comment (zengzengzenghuy) [inline docs/node/tools/dappnode.md:151]: Can you make sure that the Program already end? @alexpeterson91

Comment (alexpeterson91) [reply docs/node/tools/dappnode.md:151]: Im 100% sure . @zengzengzenghuy but thanks for double checking; original source of this page is here, and shows the end of the program. https://www.notion.so/usedappnode/Dappnode-Gnosis-Chain-Decentralized-Hardware-Validator-Incentive-Program-a62f3712ed864552a04fb72e0cb82ffb?pvs=4

Comment (zengzengzenghuy) [reply docs/node/tools/dappnode.md:151]: Got it. Thanks for confirming with me. @alexpeterson91

**Review by zengzengzenghuy — APPROVED** [review]
Thanks Alex :1st_place_medal:

**Pull request merged**